pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/NamespaceStatsAggregator.java

```diff
@@ -250,15 +250,18 @@ private static void getTopicStats(Topic topic, TopicStats stats, boolean include
         }
 
         topic.getReplicators().forEach((cluster, replicator) -> {
-            AggregatedReplicationStats aggReplStats = stats.replicationStats.computeIfAbsent(cluster,
-                    k -> new AggregatedReplicationStats());
-
             ReplicatorStatsImpl replStats = replicator.getStats();
+            AggregatedReplicationStats aggReplStats = stats.replicationStats.get(replicator.getRemoteCluster());
+            if (aggReplStats == null) {
+                aggReplStats =
+                        stats.replicationStats.put(replicator.getRemoteCluster(), new AggregatedReplicationStats());
```

Comment:
`HashMap#put` return the previous value associated with key.  Hence the return value is null